cli/dcoscli/data/help/package.txt

```diff
@@ -16,17 +21,37 @@ Usage:
 
 Options:
     --all                                  Apply the operation to all matching packages
+
     --app                                  Apply the operation only to the package's
-                                           application
+                                           marathon application
+
     --app-id=<app-id>                      The application id
+
     --cli                                  Apply the operation only to the package's CLI
+                                           command
+
+    --config                               Print the package's config.json, which contains
+                                           the configurable properties for the app
+
     -h, --help                             Show this screen
+
     --info                                 Show a short description of this subcommand
+
     --options=<file>                       Path to a JSON file containing package
                                            installation options
+
     --package-version=<package_version>    Package version to install
+
+    --package-versions                     Print all versions for this package
+
+    --render                               Render the package's marathon.json or
+                                           command.json template, rather than printing
+                                           the raw template.
```

Comment:
It's still not clear what render means here, perhaps embellish the description as follows: `Render the package's marathon.json or command.json template with the configuration values it will be installed with, rather than printing out the raw template.`